fabfile/add.py

```diff
@@ -31,6 +43,11 @@
     "./inventory.template": "../{0}/orchestration/inventory/{2}"
 }
 
+SECRETS_PARAMS = [
+    "secret_key",
```

Comment:
Is it worth prompting the user that if they're use RabbitMQ or other services with secrets that they should manually edit those in the `orchestration` folder when this is done running?